gulpfile.js

```diff
@@ -75,6 +75,70 @@ const removePrefix = (gulpRenameOptions) => {
   return gulpRenameOptions;
 };
 
+const tokenify = require('./formats/tokens');
+
+theo.registerTransform('theme', ['color/hex']);
+theo.registerFormat('light.yml', tokenify('light'));
+theo.registerFormat('dark.yml', tokenify('dark'));
+
+const colorSchemes = [
+  {transformType: 'raw', formatType: 'light.yml'},
+  {transformType: 'raw', formatType: 'dark.yml'},
+];
+
+const colorSystemFormats = [
+  {transformType: 'web/js', formatType: 'json'},
+  {transformType: 'android', formatType: 'android.xml'},
+  {transformType: 'ios', formatType: 'ios.json'},
+];
+
+gulp.task('themes', (done) => {
+  gulp
+    .src('tokens/themes/*.yml')
+    .pipe($.rename(addPrefix))
+    .pipe(
+      $.theo({
+        transform: {type: 'theme'},
+        format: {type: 'json'},
+      }),
+    )
+    .pipe($.rename(removePrefix))
+    .on('error', (err) => {
+      throw new Error(err);
+    })
+    .pipe(gulp.dest('dist'));
+  done();
+});
+
+gulp.task('palettes', (done) => {
+  colorSchemes.map(
+    ({transformType: schemeTransform, formatType: schemeFormat}) =>
+      colorSystemFormats.map(({transformType, formatType}) =>
+        gulp
+          .src('tokens/themes/*.yml')
+          .pipe(
+            $.theo({
```

Comment:
Useing theo to transform a theme into a palette, which is then passed into theo again feels a little odd to me. I wonder if we could do this with plain old JS, rather than having to register a theo transform.

That can be a problem for later though, no need to block this PR

Reply:
Leaving this comment unresolved for posterity and future PRs